runtime_constraints/README.md

```diff
@@ -0,0 +1,11 @@
+# Cluster Runtime Constraints
+
+Cluster runtime constraints are base constraints that always get applied to the resolution process to avoid installing
```

Comment:
It seems more apt to call it something like "global" or "static" than "cluster runtime", since this could be conflated with the "virtual bundle" concept mapped out in some of the EPs.

Reply:
damn - I see a lot of refactoring in my future now hahaha